client/src/modules/station/playlist/tab-playlist/data.js

```diff
@@ -0,0 +1,74 @@
+export const items = [
```

Comment:
JS ??? this should be a ts file & use the typing from operations/index.tsx

Reply:
The query for this list has been implemented, use them. And also implement the subscription for the playlist.